connect/src/main/java/com/stripe/android/connect/webview/StripeConnectWebViewClient.kt

```diff
@@ -95,7 +111,26 @@ internal class StripeConnectWebViewClient(
         }
     }
 
-    private inner class StripeJsInterfaceInternal {
+    private inner class StripeJsInterfaceInternal(private val webView: WebView) {
+        private var isPageLoaded: Boolean = false
+
+        init {
+            // Bind appearance changes in the manager to the WebView (only when page is loaded).
+            webView.doOnAttach { view ->
```

Comment:
👀 I'd appreciate extra scrutiny here to ensure this is correct.

Reply:
This looks fine to me, but why not move this to the the webview's [onPageFinished callback](https://developer.android.com/reference/android/webkit/WebViewClient#onPageFinished(android.webkit.WebView,%20java.lang.String))?

Reply:
@lng-stripe and I talked offline, `onPageFinished` wouldn't quite be right because it wouldn't capture the `onPageLoad` event